### DISCUSSION TITLE
SDK for SigmaStar SC335

### DISCUSSION THREAD

**Discussion by wildseahorse** [roleoroleo/yi-hack-MStar#388] [Q&A]
Hi folks,

Could you please share this SDK ? I can give in return SDK for Hi3518EV200 with all bundles from Hisilicon. I can't find SDK for Sigmastar SSC335, just having camera for this SoC and need this SDK.

Thanks.

**Comment by roleoroleo** [comment]
Try this:
https://drive.google.com/file/d/1DnTe8G4FV4rMuopYFrdX6sx_ZeFv9YMD/view?usp=sharing
I don't know it it works for SSC335. I'm using it with MSC313e.

**Reply by wildseahorse** [reply]
Thank you a lot, will try it with my camera, I can also give you SDK for Hi3518Ev200 and SigmaStar SDD21x/SDD20x, Do you need them ?

**Reply by roleoroleo** [reply]
Why not...

**Reply by wildseahorse** [reply]
Please try this link https://drive.google.com/drive/folders/1eR6_5okzWY_GnwLfJoRhbWLvwMMALrPH?usp=sharing

This is Hi3518eV200 SDK with hardware design manual and Orcad design files, it is little bit old, i guess it is 12 years for HI3518 still doing it's job. But i could buy it on Aliexpress for a very small price.

Let me know please if this link worked and you could open the archive ?

**Reply by roleoroleo** [reply]
Thanks, I downloaded the archive.

**Reply by reperam** [reply]
Hey I was wondering if you had a different link as the one above is dead

**Reply by roleoroleo** [reply]
Which sdk are you looking for?

**Reply by reperam** [reply]
SSC335 would be perfect, but if you happen to also have infinity6e ssc012b s01a compatible that would be amazing

**Reply by roleoroleo** [reply]
I don't have sdk for ssc335 but it should be this archive:
https://github.com/OpenIPC/firmware/releases/download/toolchain/toolchain.sigmastar-infinity6b0.tgz
